string hallHigh # filled based on hall effect sensor input
bool hallVal # debugging value for hall effect sensor input